styleguide.config.js

```diff
@@ -14,91 +14,41 @@ module.exports = {
       sections: [
         {
           name: 'Getting Started',
-          content: 'docs/introduction/getting_started.md'
+          content: 'docs/introduction/getting_started.md',
         },
         {
           name: 'Atomic Design',
-          content: 'docs/introduction/atomic_design.md'
+          content: 'docs/introduction/atomic_design.md',
         },
         {
           name: 'Component Design',
-          content: 'docs/introduction/component_design.md'
+          content: 'docs/introduction/component_design.md',
         },
         {
           name: 'Best Practices',
-          content: 'docs/introduction/best_practices.md'
+          content: 'docs/introduction/best_practices.md',
         },
       ]
     },
     {
       name: 'Colors',
-      content: 'src/atoms/Color/Colors.md'
+      content: 'src/atoms/Color/Colors.md',
     },
     {
       name: 'Atoms',
-      components: 'src/atoms/**/*.js'
+      components: 'src/atoms/**/index.js'
     },
     {
       name: 'Molecules',
-      sections: [
-        {
-          name: 'Asides',
-          components: 'src/molecules/asides/**/*.js'
-        },
-        {
-          name: 'DataRow',
-          components: 'src/molecules/DataRow/**/*.js'
-        },
-        {
-          name: 'FlashMessage',
-          components: 'src/molecules/FlashMessage/**/*.js'
-        },
-        {
-          name: 'FormFields',
-          components: 'src/molecules/formfields/**/*.js'
-        },
-        {
-          name: 'Headers',
-          components: () => [ 'src/molecules/PriceDiscountHeader/index.js' ]
-        },
-        {
-          name: 'Lists',
-          components: 'src/molecules/lists/**/*.js'
-        },
-        {
-          name: 'Modal',
-          components: 'src/molecules/Modal/**/*.js'
-        },
-        {
-          name: 'TextLockups',
-          components: 'src/molecules/TextLockup/**/*.js'
-        },
-        {
-          name: 'Other',
-          components: () => [ 'src/molecules/StepProgress/index.js' ]
-        }
-      ]
+      components: 'src/molecules/**/index.js',
```

Comment:
This removes the sectioning on things like `FormFields` and `TextLockups`. These were there on purpose.